src/components/Popups/PopupBase.scss

```diff
@@ -2,13 +2,25 @@
 
 .ba {
     $popup-arrow-size: 12px;
+    $popup-button-border-radius: 6px;
```

Comment:
Does this need to be a variable if it's only used once? Also seems like it should match the default border radius value from BUIE. Can you check with design?

Reply:
Sure, according to the Zeplin it is different but I'll check.